Proxies/Array.cs

```diff
@@ -21,25 +21,43 @@ public abstract class ArrayProxy {
         [JSAlwaysAccessAsProperty]
         [JSNeverReplace]
         [JSIsPure]
-        abstract public long LongLength { get; } 
+        abstract public long LongLength { get; }
 
 
+        [JSReplacement("$$jsilcore.System.Array.prototype.GetLowerBound.call($this, $dimenstion)")]
+        public int GetLowerBound(int dimenstion)
+        {
+            throw new InvalidOperationException();
+        }
+
+        [JSReplacement("$$jsilcore.System.Array.prototype.GetUpperBound.call($this, $dimenstion)")]
+        public int GetUpperBound(int dimenstion)
+        {
+            throw new InvalidOperationException();
+        }
+
+        [JSReplacement("$$jsilcore.System.Array.prototype.GetLength.call($this, $dimenstion)")]
+        public int GetLength(int dimenstion)
+        {
+            throw new InvalidOperationException();
+        }
+
         [JSReplacement("JSIL.Array.New($elementType, $size)")]
         public static System.Array CreateInstance (Type elementType, Int32 size) {
             throw new InvalidOperationException();
         }
 
-        [JSReplacement("JSIL.MultidimensionalArray.New.apply(null, [$elementType].concat($sizes))")]
+        [JSReplacement("JSIL.MultidimensionalArray.CreateInstance(elementType, $sizes)")]
```

Comment:
Really, I tried to mimic .Net method signatures.
Array type ctor (New):
A constructor that takes twice as many int32 arguments as there are dimensions of the
array. These arguments occur in pairs—one pair per dimension—with the first argument of
each pair specifying the lower bound for that dimension, and the second argument 

Array.CreateInstance:
Creates a multidimensional Array of the specified Type and dimension lengths, with zero-based indexing. The dimension lengths are specified in an array of 32-bit integers.

Could you suggest better names?